index.html

```diff
@@ -40,6 +41,188 @@ <h3 class="subject_text">I'm a Software Developer</h3>
+            <div class="project_title">
+                <h3>Trading Website</h3>
+            </div>
+            <div class="project_description">
+                <h4>A daily selection of privately personalized reads; 
+                no accounts or sign-ups required. Has been the 
+                industry's standard.</h4>
+            </div>
+            <div class="project_image"></div>
+            <ul class="project_skills">
+                <li class="skill_container"><h4 class="skill">HTML</h4></li>
+                <li class="skill_container"><h4 class="skill">Bootstrap</h4></li>
+                <li class="skill_container"><h4 class="skill">Ruby</h4></li>
+            </ul>
+            <button class="project_link"><h4 class="link_text">See project</h4></button>
+        </div>
+    </section>
+    <section class="about_me">
+        <div class="bio_header">
+            <h1>About me</h1>
+        </div>
+        <div class="bio">
+            <h4>Hello I'm a software developer! I can help you build a product , feature or website 
+            Look through some of my work and experience! If you like what you see and have a 
+            project you need coded, don't hesitate to contact me.</h4>
+        </div>
+
+        <a href="#" class="resume_link">
+            <button class="resume_button">
+                <h4 class="link_text">Get my resume</h4>
+            </button>
+        </a>
+        <div class="about_me_illustration"></div>
+        <div class="separation_line">
+            <hr>
+        </div>
+        <div class="knowledge">
+            <div class="competent">
+                <div class="category_style">
+                    <div class="bullet_1"></div>
+                    <h3 class="category">Language</h3>
+                </div>
+                <h4 class="competency">Javascript</h4>
+                <h4 class="competency">Ruby</h4>
+                <h4 class="competency">HTML</h4>
+                <h4 class="competency">CSS</h4>
+            </div>
+            <div class="competent">
+                <div class="category_style">
+                    <div class="bullet_2"></div>
+                    <h3 class="category">Frameworks</h3>
+                </div>
+                <h4 class="competency">React.js</h4>
+                <h4 class="competency">Ruby on Rails</h4>
+                <h4 class="competency">RSpec</h4>
+                <h4 class="competency">Capybara</h4>
+            </div>
+            <div class="competent">
+                <div class="category_style">
+                    <div class="bullet_3"></div>
+                    <h3 class="category">Skills</h3>
+                </div>
+                <h4 class="competency">Database Management</h4>
+                <h4 class="competency">Version Control</h4>
+                <h4 class="competency">CLI</h4>
+                <h4 class="competency">API Design</h4>
+                <h4 class="competency">Web Development</h4>
+            </div>
```

Comment:
- Kindly, make sure you use the `ul` tag to list the elements on this block instead of using the `div` tag, and note that each should be inside a `li` tag.😄

According to the Figma design, I suggest you fix some margins and padding to make your project match the Figma Design.

> ## Your Design
>
><img width="372" alt="Screenshot 2023-06-03 at 5 52 15 PM" src="https://github.com/lewisjn-arch/Mobile-Portfolio-setup/assets/91237525/d827ec28-30b2-4d87-8046-4cbe7634ca7c">
>
> ## The Figma Design
>
><img width="465" alt="Screenshot 2023-06-03 at 5 53 13 PM" src="https://github.com/lewisjn-arch/Mobile-Portfolio-setup/assets/91237525/7ca71f6e-068a-40d8-8618-d5c615eba6f7">

Reply:
@stanleySimeon  Thank you so much for your review, let me get to it right now as I was unwell with a flu hence I am a bit late with the project.